app.py

```diff
@@ -19,7 +19,7 @@
     update_or_create_criterion,
     update_or_create_link,
 )
-from models import Category, Criterion, Link  # noqa: E402
+from models import Category, Criterion, Link, Score  # noqa: E402
```

Comment:
I think this got lost in all the PRs that went in this weekend! This should fix this failing commit: https://github.com/RagtagOpen/freefrom-map-backend/commit/3c702305cb732b95bbfff01b9c305950eb552396

Reply:
Oh I made this change because of flake8 but it must have broken when it was merged with your Score PR. Sorry about that!

Reply:
Don't even worry! I should've noticed it failed before merging :)